gen-ai/orchestrator-server/src/main/python/server/src/gen_ai_orchestrator/routers/requests/requests.py

```diff
@@ -52,7 +54,61 @@ class ObservabilityProviderSettingStatusQuery(BaseModel):
     setting: ObservabilitySetting = Field(description='The Observability Provider setting to be checked.')
 
 
-class RagQuery(BaseModel):
+class QAQuery(BaseModel):
+    """The QA query model"""
+
+    embedding_question_em_setting: EMSetting = Field(
+        description="Embedding model setting, used to calculate the user's question vector."
+    )
+    document_index_name: str = Field(
+        description='Index name corresponding to a document collection in the vector database.',
+    )
+    document_search_params: DocumentSearchParams = Field(
+        description='The document search parameters. Ex: number of documents, metadata filter',
+    )
+    question_answering_prompt_inputs: Any = Field(
```

Comment:
I agree with you. We used to have a prompt in our use cases that added a few pieces of information with the user request, but this can be added directly on the client side.